xml/System.Windows.Forms/OpenFileDialog.xml

```diff
@@ -47,8 +47,8 @@
 On a right-to-left operating system, setting the containing form's <xref:System.Windows.Forms.Control.RightToLeft> property to <xref:System.Windows.Forms.RightToLeft.Yes?displayProperty=nameWithType> localizes the dialog's **File Name**, **Open**, and **Cancel** buttons. If the property is not set to <xref:System.Windows.Forms.RightToLeft.Yes?displayProperty=nameWithType>, English text is used instead.  
 
  If you want to give the user the ability to select a folder instead of a file, use <xref:System.Windows.Forms.FolderBrowserDialog> instead.  
-  
-   
```

Comment:
This should be documented as an `<exception>` in the member, not in the type.
https://github.com/dotnet/dotnet-api-docs/blob/5b0810324b50815313c14a862ab4acc2290c9243/xml/System.Windows.Forms/OpenFileDialog.xml#L237

For reference, for [System.IO.File.Open](https://learn.microsoft.com/en-us/dotnet/api/system.io.file.open?view=net-8.0) we say "An I/O error occurred while opening the file" for `IOException`s.

Reply:
Thank you for information. I did as you suggested :)